accord-core/src/main/java/accord/coordinate/RecoverWithRoute.java

```diff
@@ -211,9 +211,15 @@ else if (!known.definition.isOrWasKnown())
                 if (known.executeAt.isDecidedAndKnownToExecute() && known.deps.hasDecidedDeps() && known.outcome == Apply)
                 {
                     Deps deps = full.stableDeps.reconstitute(route());
-                    node.withEpoch(full.executeAt.epoch(), () -> {
+                    node.withEpoch(full.executeAt.epoch(), (ignored, withEpochFailure) -> {
+                        if (withEpochFailure != null)
+                        {
+                            // TODO (review): We already ignore persist errors here by not providing a callback to persist
+                            return;
+                        }
                         persist(node.coordinationAdapter(txnId, InitiateRecovery), node, topologies, route(), txnId, txn, full.executeAt, deps, full.writes, full.result, null);
                     });
+                    // TODO (review): We say it was applied regardless of what happens elsewhere? Could/should this move into the withEpoch?
                     callback.accept(APPLIED, null);
```

Comment:
@belliottsmith I want to call this out. It seems like we assume it's applied without checking if it actually worked?